assets/stylesheets/screens/_keyword.scss

```
@@ -20,6 +20,30 @@ body.keyword.post {
         @apply w-1/4 m-0 ml-4;
       }
     }
+
+    .form-keyword-search {
+      @apply mt-6 border-t border-gray-200;
+
+      &__title {
+        @apply text-lg font-semibold mt-4;
+      }
+
+      &__content {
+        @apply w-full flex flex-row mt-4;
+      }
+
+      &__input-keyword {
+        @apply w-3/4 m-0;
+      }
+
+      &__btn-search {
+        @apply w-1/4 m-0 ml-4;
+      }
+    }
+
+    .no-keyword {
```

In terms of naming, to make this more generic, one can use the concert of `blank-state`. Designers also refer to the UI when there is no data as blank state.

```suggestion
    .blank-state {
```